lib/jekyll-compose/arg_parser.rb

```diff
@@ -24,4 +25,8 @@ def title
   def force?
     !!options["force"]
   end
+
+  def source
+    source = config['source'].gsub(/^#{Regexp.quote(Dir.pwd)}/, '')
```

Comment:
Other methods had an assign, so I thought I would follow that pattern. I can remove this if it isn't doing anything.